magento/pwa-studio
# magento/pwa-studio#: Fix links in the “Contributing” section

## Description
This pull request fixes links in the "Contributing" section of https://github.com/magento/pwa-studio/tree/develop/pwa-devdocs page.

## Related Issue





Closes #ISSUE_NUMBER.

## Verification Steps

1. Go to https://github.com/magento/pwa-studio/tree/develop/pwa-devdocs page.
2. Check `contribution guidelines` and `code of conduct` links.

**Actual result:** mentioned links go to `404` page.
**Expected result:** links should contain correct URLs


## Proposed Labels for Change Type/Package




## Checklist:



Thank you!
